### PR TITLE
[IOSP-226] App Center invitation instructions

### PR DESCRIPTION
**Pull request by MichalTKwiecien**
This section describes how we can invite new people to the organisation.

### CODE REVIEW

File: Cookbook/Technical-Documents/AppCenter.md [hunk 1]
```diff
@@ -59,3 +61,21 @@ If this happened, follow these steps:
...
+
+1. Visit [App Center login page](https://appcenter.ms/sign-in) (using incognito mode to avoid logging you out from your account)
+
+2. Click `Sign in with email` on the bottom of the page.
+
+3. Use `App Center` credentials from iOS Team 1Password.
+
+4. On the left panel, select `Babylon Health` organization and go to the `People` page.
+
+5. Inside the `Collaborators` tab, invite someone using his/her `@babylonhealth.com` email.
```
Comment (brocoo) [inline Cookbook/Technical-Documents/AppCenter.md]:
```suggestion
5. Inside the `Collaborators` tab, invite someone using their `@babylonhealth.com` email.
```

File: Cookbook/Technical-Documents/AppCenter.md [hunk 2]
```diff
@@ -59,3 +61,21 @@ If this happened, follow these steps:
...
+
+6. Switch to the `Teams` tab and select the team you want to add the person to. "BABYLON-TEAM" gives access to download builds for all of our projects so if you're not sure which one you should select, just invite there.
+
+7. Inside the team, add collaborator using his/her email (**Unfortunately, you can only add people that already accepted an invitation to the organisation so you need to wait for them to accept**).
```
Comment (brocoo) [inline Cookbook/Technical-Documents/AppCenter.md]:
```suggestion
7. Inside the team, add the collaborator using their email (**Unfortunately, you can only add people that already accepted an invitation to the organisation so you need to wait for them to accept**).
```

File: Cookbook/Technical-Documents/AppCenter.md (hunk 1 above)
Comment (sdaniel55) [inline Cookbook/Technical-Documents/AppCenter.md]:
```suggestion
5. Inside the `Collaborators` tab, invite someone using their `@babylonhealth.com` email.
```

**Review by brocoo — APPROVED** [review]
Just a few nits, but otherwise 👍

File: Cookbook/Technical-Documents/AppCenter.md [hunk 3]
```diff
@@ -59,3 +61,21 @@ If this happened, follow these steps:
...
+
+2. Click `Sign in with email` on the bottom of the page.
+
+3. Use `App Center` credentials from iOS Team 1Password.
+
+4. On the left panel, select `Babylon Health` organization and go to the `People` page.
+
+5. Inside the `Collaborators` tab, invite someone using his/her `@babylonhealth.com` email.
+
+6. Switch to the `Teams` tab and select the team you want to add the person to. "BABYLON-TEAM" gives access to download builds for all of our projects so if you're not sure which one you should select, just invite there.
```
Comment (sdaniel55) [inline Cookbook/Technical-Documents/AppCenter.md]:
```suggestion
6. Switch to the `Teams` tab and select the team you want to add the person to. `BABYLON-TEAM` gives access to download builds for all of our projects so if you're not sure which one you should select, just invite there.
```

File: Cookbook/Technical-Documents/AppCenter.md (hunk 2 above)
Comment (sdaniel55) [inline Cookbook/Technical-Documents/AppCenter.md]:
```suggestion
7. Inside the team, add collaborator using their email (**Unfortunately, you can only add people that already accepted an invitation to the organisation so you need to wait for them to accept**).
```

**Review by danspndl — APPROVED** [review]
Left some comments 🚀

**Comment by MichalTKwiecien** [comment]
After investigating the problem that someone just posted on #ios slack channel, I've added one new point to the instructions in https://github.com/Babylonpartners/ios-playbook/pull/259/commits/34cb3726fd2fcd738f9eeb0513460345efc26f45

**Pull request merged**